packages/server/src/services/EmailService.ts

```diff
@@ -24,16 +25,18 @@ export default class EmailService extends BaseService {
 				if (!this.senderInfo(EmailSender.NoReply).email) {
 					throw new Error('No-reply email must be set for email service to work (Set env variable MAILER_NOREPLY_EMAIL)');
 				}
-
-				this.transport_ = createTransport({
+				const options: SMTPTransport.Options = {
 					host: this.config.mailer.host,
 					port: this.config.mailer.port,
 					secure: this.config.mailer.secure,
-					auth: {
+				};
+				if (this.config.mailer.authUser !== '' && this.config.mailer.authPassword !== '') {
```

Comment:
`this.config.mailer.authUser || this.config.mailer.authPassword`

Reply:
Yes looks good, your suggestion looks a lot cleaner. It changes the behavior a little because it tries to do authentication if only one of the two required environment variables are set. But,he, if you set one, you probably want to set the other too.
I changed it as suggested.